build.gradle

```diff
@@ -1,17 +1,16 @@
 buildscript {
-    ext {
-        springBootVersion = "1.2.1.RELEASE"
-    }
     repositories {
         mavenLocal()
         jcenter()
         mavenCentral()
         maven { url "http://repo.spring.io/libs-snapshot" }
+        maven { url 'https://oss.sonatype.org/content/groups/public/' }

```

Comment:
Ok, probably it is for Accurest. Nevertheless `http://repo.spring.io/libs-snapshot` could be probably removed.